### PR TITLE
Html tag processing

### PR DESCRIPTION
**Pull request by maklyakov**

### CODE REVIEW

**Comment by maklyakov** [comment]
[Cookie Kit - order of processing seems wrong](https://trello.com/c/WZqn5FPN/1829-cookie-kit-order-of-processing-seems-wrong)

**Comment by maklyakov** [comment]
[Add automatic HTML tag processing to cookie-kit](https://trello.com/c/JNH9Y6hF/1830-add-automatic-html-tag-processing-to-cookie-kit)

**Comment by maklyakov** [comment]
[Add automatic HTML tag processing to cookie-kit](https://trello.com/c/JNH9Y6hF/1830-add-automatic-html-tag-processing-to-cookie-kit)

File: packages/xcoobee-cookie-kit-web/src/devOnly.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import "./index";
 window.theCookieHandler = (cookieConsents) => {
```
Comment (Bilal-S) [inline packages/xcoobee-cookie-kit-web/src/devOnly.js]: We should probably exclude this file from the distribution in `.gitignore`

**Pull request merged**